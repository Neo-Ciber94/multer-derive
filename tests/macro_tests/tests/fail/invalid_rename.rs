use multer_derive::FromMultipart;

#[derive(FromMultipart)]
struct MyStruct {
    #[multer(rename = "1")]
    text: String,

    #[multer(rename = "$number")]
    number: u32,
}

fn main() {}
