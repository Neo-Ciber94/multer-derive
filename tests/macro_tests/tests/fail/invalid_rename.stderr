error: proc-macro derive panicked
 --> tests/fail/invalid_rename.rs:3:10
  |
3 | #[derive(FromMultipart)]
  |          ^^^^^^^^^^^^^
  |
  = help: message: `"1"` is not a valid identifier
